go.mod

```diff
@@ -13,3 +13,65 @@ require (
 	github.com/hashicorp/terraform-svchost v0.0.0-20191011084731-65d371908596
 	golang.org/x/time v0.0.0-20210611083556-38a9dc6acbc6 // indirect
 )
+
+require (
```

Comment:
Per release notes

> Because the number of explicit requirements may be substantially larger in an expanded Go 1.17 go.mod file, the newly-added requirements on indirect dependencies in a go 1.17 module are maintained in a separate require block from the block containing direct dependencies.